src/test/blk_non_zero/TEST11

```diff
@@ -1,37 +1,9 @@
 #!/usr/bin/env bash
 #
+# SPDX-License-Identifier: BSD-3-Clause
 # Copyright 2015-2019, Intel Corporation
 # Copyright (c) 2016, Microsoft Corporation. All rights reserved.
 # Copyright 2019, IBM Corporation
-#
```

Comment:
This commit is not supposed to remove any license text, but it is removed from this file and few others.

Reply:
All such files were Intel+Microsoft+IBM.

Reply:
This doesn't explain the discrepancy between the commit description and the commit body.